lib/common/ipc_server.c

```diff
@@ -174,7 +174,10 @@ client_from_connection(qb_ipcs_connection_t *c, void *key, uid_t uid_client)
         client->user = pcmk__uid2username(uid_client);
         if (client->user == NULL) {
             client->user = strdup("#unprivileged");
-            CRM_CHECK(client->user != NULL, free(client); return NULL);
+            if (client->user == NULL) {
+                free(client);
+                return NULL;
+            }
```

Comment:
This is only used by daemons, so we should just assert the allocations in this function

Reply:
Sounds good. I saw that it was used in at least one daemon IPC callback (`attrd_ipc_accept()`) and didn't feel like thinking about whether that might affect the client somehow.